radioactivedecay/inventory.py

```diff
@@ -272,19 +533,85 @@ def radionuclides(self) -> List[str]:
 
         return list(self.contents)
 
-    @property
-    def activities(self) -> List[float]:
+    def numbers(self) -> Dict[str, float]:
         """
-        Returns a list of the activities of the radionuclides in the Inventory.
+        Returns a dictionary of radionuclides and associated numbers in the inventory.
 
         Examples
         --------
-        >>> rd.Inventory({'Tc-99m': 2.3, 'I-123': 5.8}).activities
-        [5.8, 2.3]
+        >>> rd.Inventory({'Tc-99m': 2.3, 'I-123': 5.8}).numbers()
+        {'I-123': 399738.47946141585, 'Tc-99m': 71852.27235544211}
 
         """
 
-        return list(self.contents.values())
+        return self.contents
+
+    def activities(self) -> Dict[str, float]:
+        """
+        Returns a dictionary of radionuclides and associated activities in the inventory.
```

Comment:
Maybe "Returns a dictionary containing the activity of each nuclide within the inventory." is a clearer way to put this. Also for the other methods below.